site/src/pages/UserSettingsPage/SSHKeysPage/SSHKeysPage.tsx

```diff
@@ -7,8 +7,14 @@ import { SSHKeysPageView } from "./SSHKeysPageView"
 
 export const Language = {
   title: "SSH keys",
-  description:
-    "Coder automatically inserts a private key into every workspace; you can add the corresponding public key to any services (such as Git) that you need access to from your workspace.",
+  description: (
+    <p>
+      Git in workspaces is authenticated with the following public key. You may add it to Git
+      services (such as GitHub) that you need to access from your workspace. <br />
+      <br />
+      Coder sets <code>$GIT_SSH_COMMAND</code> to perform the override.
```

Comment:
You haven't referenced an override yet so this could be more clear, like "to override ___"